### PR TITLE
[12.0] Add shopinvader_notification_default

### PR DESCRIPTION
**Pull request by simahawk**
Implements #496.
Depends on https://github.com/OCA/social/pull/471
Needs: official default email texts.

@Cedric-Pigeon maybe you can help me here w/ some email text you are currently using for your customers?

### CODE REVIEW

File: oca_dependencies.txt [hunk 1]
```diff
@@ -8,6 +8,8 @@ search-engine
 product-attribute
 product-variant
 sale-workflow
+#social
+social https://github.com/camptocamp/social 12-mail_inline_css
```
Comment (lmignon) [inline oca_dependencies.txt]: merged ...

**Review by lmignon — APPROVED** [review]
LGTM (code review)

PR on social  is now merged. Can you adapt the oca_dependencies before merging PLZ?

**Comment by simahawk** [comment]
@lmignon I'm not sure we want to merge in current state: there's no content in the emails. I'd like to have some simple but good text content to put in all of them.
I was actually waiting for ppl from acsone or akretion to provide me those texts as you have already something in place I guess :wink: 
That's why WIP label is there.

**Comment by simahawk** [comment]
giving precedence to v13 https://github.com/shopinvader/odoo-shopinvader/pull/544

**Pull request closed without merging**